data/changelog/opam/2018-07-26-opam-2-0-0-rc4.md

```diff
@@ -10,9 +10,9 @@ We are happy to announce the [opam 2.0.0 final release candidate](https://github
 
 This release features a few bugfixes over [Release Candidate 3](../opam-2-0-0-rc3). **It will be promoted to 2.0.0 proper within a few weeks, when the [official repository](https://github.com/ocaml/opam-repository) format switches from 1.2.0 to 2.0.0.** After that date, updates to the 1.2.0 repository may become limited, as new features are getting used in packages.
 
-It is safe to update as soon as you see fit, since opam 2.0.0 supports the older formats. See the [Upgrade Guide](https://opam.ocaml.org/2.0-preview/doc/Upgrade_guide.html) for details about the new features and changes. If you are a package maintainer, you should keep publishing as before for now: the [roadmap](https://opam.ocaml.org/blog/opam-2-0-0-repo-upgrade-roadmap) for the repository upgrade will be detailed shortly.
+It is safe to update as soon as you see fit, since opam 2.0.0 supports the older formats. See the [Upgrade Guide](https://opam.ocaml.org/2.0-preview/doc/Upgrade_guide.html - [404 Not Found]) for details about the new features and changes. If you are a package maintainer, you should keep publishing as before for now: the [roadmap](https://opam.ocaml.org/blog/opam-2-0-0-repo-upgrade-roadmap) for the repository upgrade will be detailed shortly.
```

Comment:
```suggestion
It is safe to update as soon as you see fit, since opam 2.0.0 supports the older formats. See the [Upgrade Guide](https://opam.ocaml.org/doc/Upgrade_guide.html) for details about the new features and changes. If you are a package maintainer, you should keep publishing as before for now: the [roadmap](https://opam.ocaml.org/blog/opam-2-0-0-repo-upgrade-roadmap) for the repository upgrade will be detailed shortly.
```